apps/emqx/src/emqx_schema.erl

```diff
@@ -1900,7 +1903,8 @@ common_ssl_opts_schema(Defaults) ->
                     required => false,
                     example => <<"">>,
                     format => <<"password">>,
-                    desc => ?DESC(common_ssl_opts_schema_password)
+                    desc => ?DESC(common_ssl_opts_schema_password),
+                    converter => fun password_converter/2
```

Comment:
if the db_name also needs to use this function, maybe the name `try_converter_to_binary` will be better?

Reply:
if it's sensitive information, it'd be better to use the function name with a sensitive keyword.

Reply:
I mean, for db_name, is it possible to use integer?

Reply:
added another export: `bin_str_cvonverter/2`